rasterio/merge.py

```diff
@@ -7,6 +7,12 @@
 
 import numpy as np
 
+try:
+    from pathlib import Path
+except ImportError:  # pragma: no cover
+    class Path:
+        pass
+
```

Comment:
@fwfichtner you can delete this because in version 1.2.0 (to be released from the master branch) we will only support Python versions with pathlib.